xbmc/dialogs/GUIDialogKaiToast.cpp

```diff
@@ -134,6 +135,9 @@ bool CGUIDialogKaiToast::DoWork()
     //  Play the window specific init sound for each notification queued
     SetSound(toast.withSound);
 
+    // Activate haptics for this notification
+    PERIPHERALS::g_peripherals.OnUserNotification();
```

Comment:
I'd have to take a closer look at the code myself.

It's called "Kai Toast" and not "Kia Toast"